site/content/docs/traces-open-telemetry/exporting-traces/dash0.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Exporting traces to Dash0 - Checkly Docs
+displayTitle: Exporting traces to Dash0
+navTitle: To Dash0
+description: "Export traces from Checkly to Dash0 for better observability."
+weight: 24
+menu:
+  platform:
+    parent: Export traces to 3rd party tools with Checkly - Checkly Docs
+beta: true
+aliases:
+  - "/docs/open-telemetry/exporting-traces/dash0/"
```

Comment:
```suggestion
```

We can remove the alias here, since that's an old redirect of the docs that Dash0 won't need